apps/console/src/features/applications/components/settings/attribute-management/attribute-settings.tsx

```diff
@@ -560,6 +562,8 @@ export const AttributeSettings: FunctionComponent<AttributeSelectionPropsInterfa
     };
 
     const updateValues = () => {
+        eventPublisher.publish("application-user-attribute-click-update-button");
+        
```

Comment:
Redundant line.

Reply:
This place has changed with some other code. Hence didn't remove the extra line